### ISSUE TITLE
Trace zips not viewable

### ISSUE THREAD

**Issue by RichardGuion** [alexneo2003/playwright-azure-reporter#5]
This may be more of a Playwright issue, but I thought I would ask here to see if anyone else had it as well:

I've been running playwright on ADO, and sending test result pass/fail back to ADO test cases with the playwright-azure-reporter
It is all working fine, and storing the screenshots, videos and trace files on the ADO test result. I set it to save this stuff in the config file:

`attachmentsType: ['screenshot', 'video', 'trace'],`

However, when I try to download a trace zip (from the ADO test case that has the execution result) and open it in the trace viewer (npx playwright show-trace), the viewer says it is not a valid trace file!

I've unzipped the file and tried to compare it against a trace zip produced locally on my computer - I don't see any difference between the two. The resources folder on the failing one has css, json, jpeg, etc files just like the first one does.

**Comment by alexneo2003** [comment]
i check some cases and see the next issue
in Safari trace file is downloaded as folder and if you pack it back in zip archive manually and try to open in trace-viewer library what used playwright trace viewer produce error `Error: Compression method not supported`
> open DevTools in trace viewer and see console tab

I'm at now run test and download trace file from ADO - it will downloaded as .zip file on all works fine - it opened as well
can you check downloading file in deferent browsers and let me know the result?

**Comment by RichardGuion** [comment]
You analysis was correct. When I downloaded the trace zips on Safari, it unzipped them and when I recompressed, it failed to open in trace viewer - the error in the console was about a JSON file incorrectly formatted.

I switched to Chrome, it download the zip file intact without unzipping, and I can open those fine in the trace viewer. That works for me, I will just use Chrome with ADO Test Plan to download traces from now on.
